.github/ISSUE_TEMPLATE/improve-existing-docs.yml

```
@@ -0,0 +1,62 @@
+name: Improve existing docs
+about: Make a suggestion to improve our existing documentation.
+labels: content
+inputs:
+- type: description
+  attributes:
+    value: |
+    HUBBERS BEWARE! THE GITHUB/DOCS REPO IS PUBLIC TO THE ENTIRE INTERNET. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.
+
+    For questions, ask in Discussions: https://github.com/github/docs/discussions'
```

I believe these need to be indented one more level to work properly:

```suggestion
    value: |
      HUBBERS BEWARE! THE GITHUB/DOCS REPO IS PUBLIC TO THE ENTIRE INTERNET. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.

      For questions, ask in Discussions: https://github.com/github/docs/discussions'
```